monai/auto3dseg/analyzer.py

```diff
@@ -465,16 +455,7 @@ def __call__(self, data):
         ndas_label = d[self.label_key]  # (H,W,D)
 
         if ndas_label.shape != ndas[0].shape:
-            # if image and label shapes are different, check if they are close
-            if np.allclose(list(ndas_label.shape), list(ndas[0].shape), atol=10):
-                logger.info(f" Label shape {ndas_label.shape} is slightly different from image shape {ndas[0].shape}")
-                ndas_label = F.interpolate(
-                    input=ndas_label.unsqueeze(0).unsqueeze(0),
-                    size=list(ndas[0].shape),
-                    mode="nearest-exact" if pytorch_after(1, 11) else "nearest",
-                )[0, 0]
-            else:
-                raise ValueError(f"Label shape {ndas_label.shape} is  different from image shape {ndas[0].shape}")
+            raise ValueError(f"Label shape {ndas_label.shape} is  different from image shape {ndas[0].shape}")
```

Comment:
There are two spaces between "is" and "different" here

Reply:
ok